### ISSUE TITLE
dlopen(libmediainfo.dylib, 6): image not found

### ISSUE THREAD

**Issue by OmgImAlexis** [pymedusa/Medusa#1483]
No clue what caused this. @p0psicles 

```
2016-11-06 18:07:04 ERROR    SHOWQUEUE-REFRESH :: [d36e50c] 295685: Error while refreshing show Lucifer. Error: dlopen(libmediainfo.dylib, 6): image not found
Traceback (most recent call last):
  File "/Users/**********/code/Medusa/medusa/show_queue.py", line 567, in run
    self.show.refresh_dir()
  File "/Users/**********/code/Medusa/medusa/tv.py", line 1312, in refresh_dir
    self.load_episodes_from_dir()
  File "/Users/**********/code/Medusa/medusa/tv.py", line 628, in load_episodes_from_dir
    cur_episode = self.make_ep_from_file(os.path.join(self.location, media_file))
  File "/Users/**********/code/Medusa/medusa/tv.py", line 932, in make_ep_from_file
    new_quality = Quality.nameQuality(filepath, self.is_anime)
  File "/Users/**********/code/Medusa/medusa/common.py", line 301, in nameQuality
    return Quality._extend_quality(name)
  File "/Users/**********/code/Medusa/medusa/common.py", line 399, in _extend_quality
    quality = Quality.qualityFromFileMeta(file_path)
  File "/Users/**********/code/Medusa/medusa/common.py", line 420, in qualityFromFileMeta
    knowledge = knowit.know(file_path)
  File "/Users/**********/code/Medusa/lib/knowit/api.py", line 31, in know
    return provider.describe(video_path, options)
  File "/Users/**********/code/Medusa/lib/knowit/providers/mediainfo.py", line 143, in describe
    data = MediaInfo.parse(video_path).to_data()
  File "/Users/**********/code/Medusa/lib/pymediainfo/__init__.py", line 75, in parse
    lib = CDLL("libmediainfo.dylib")
  File "/usr/local/Cellar/python/2.7.12_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/ctypes/__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(libmediainfo.dylib, 6): image not found
```

**Comment by OmgImAlexis** [comment]
Looks like every time I refresh the show it does this.

**Comment by ratoaq2** [comment]
Fixed by #1492
